lab_1/main.py

```diff
@@ -105,7 +184,11 @@ def write_to_file(path_to_file: str, content: list):
     """
     Writes the result in a file
     """
-    pass
+    if isinstance(path_to_file, str) and isinstance(content, list):
```

Comment:
Usually, there are checks for arguments and body function - separated and on the same level.
Try to get rid of nesting.